snappy_pipeline/workflows/wgs_sv_annotation/__init__.py

```diff
@@ -85,12 +85,10 @@
     path_ngs_mapping: ../ngs_mapping
     path_variant_calling: ../variant_calling
     path_wgs_sv_calling: ../wgs_sv_calling
-    tool_ngs_mapping_variant_calling: bwa
-    tool_variant_calling: gatk_hc
-    tools_ngs_mapping:
-    - bwa
-    tools_wgs_sv_calling:
-    - delly2
+    tool_ngs_mapping_variant_calling: bwa  # REQUIRED
+    tool_variant_calling: gatk_hc          # REQUIRED
+    tools_ngs_mapping: [bwa]               # REQUIRED
+    tools_wgs_sv_calling: [delly]          # REQUIRED
```

Comment:
Just one minor thing: `delly2` to `delly` is what you want, right?

Reply:
I initially didn't want to change the defaults, but it is a good suggestion. I also double checked and the last genome batch used `delly2`.